src/common/libkvs/Makefile.am

```diff
@@ -61,7 +61,6 @@ test_ldadd = \
 	$(top_builddir)/src/common/libflux/libflux.la \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libtap/libtap.la \
-	$(JANSSON_LIBS) \
```

Comment:
jansson is used in libkvs/kvs_commit.c so presumably we should keep this and add missing `$(JANSSON_CFLAGS)`.

The jansson header is included in kvs.c, but I think it could be removed.